rhba-dev-env/vagrant/provisioning/roles/setup_env/tasks/main.yml

```diff
@@ -0,0 +1,207 @@
+  become: true
+  become_user: root
+  dnf:
+    name: docker
+    state: latest
+
+- name: "docker storage configurations"
+  become: true
+  become_user: root
+  copy:
+    dest: /etc/sysconfig/docker-storage-setup
+    content: |
+          DEVS=/dev/sdb
+          VG=docker-vg
+
+- name: "docker storage setup"
+  become: true
+  become_user: root
+  command: docker-storage-setup 
+
+- name: "enable docker service"
+  become: true
+  become_user: root
+  service:
+   name: docker
+   enabled: yes
+
+- name: "start docker service"
+  become: true
+  become_user: root
+  service:
+   name: docker
+   state: started
+
+- name: ensure group "docker" exists
+  become: true
+  become_user: root
+  group:
+    name: docker
+    state: present
+
+- name: adding user vagrant to group docker
+  become: true
+  become_user: root
+  user:
+    name: vagrant
+    groups: docker
+    append: yes
+
+- name: add new docker configuration
+  become: true
+  become_user: root
+  template:
+    src: "registries.j2"
+    dest: /etc/containers/registries.conf
+
+- name: enable @cekit/cekit
+  become: true
+  become_user: root
+  command: dnf copr enable @cekit/cekit -y
+  args:
+    warn: false
+
+- name: install the latest version of cekit packages
+  become: true
+  become_user: root
+  dnf:
+    name:
+      - cekit
+      - cekit-bash-completion
+      - python3-docker
+      - python3-behave
+      - python3-lxml
+      - python3-docker-squash
+      - odcs-client
+      - krb5-workstation
+      - fedpkg
+    state: latest
+
+- name: install useful packages
+  become: true
+  become_user: root
+  dnf:
+    name:
+      - vim
+      - wget
+      - git
+    state: latest
+
+- name: create cekit config directory
+  file:
+    path: "{{ansible_user_dir}}/.cekit"
+    state: directory
+
+- name: create cekit config
+  template:
+    src: "cekit.j2"
+    dest: "{{ansible_user_dir}}/.cekit/config"
+
+- name: create koji config directory
+  file:
+    path: "{{ansible_user_dir}}/.koji"
+    state: directory
+
+- name: create koji config
+  template:
+    src: "koji.j2"
+    dest: "{{ansible_user_dir}}/.koji/config"    
+
+- name: create kerberos configuration
+  become: true
+  become_user: root
+  template:
+    src: "krb5.j2"
+    dest: /etc/krb5.conf
+    
+- name: install the latest version of ansible
+  become: true
+  become_user: root
+  dnf:
+    name: ansible
+    state: latest
+
+- name: increase ansible timeout
+  become: true
+  become_user: root
+  lineinfile:
+    path: /etc/ansible/ansible.cfg
+    regexp: "^#timeout = 10"
+    line: "timeout = 60"
+
+- name: generate ssh keys
+  openssh_keypair:
+    path: "/home/vagrant/.ssh/id_rsa"
+    size: 2048
+
+- name: creates workspace directory
+  file:
+    path: "{{ ansible_user_dir }}/workspace"
+    state: directory
+
+- name: copy rpm redhat-internal-cert
+  become: true
+  become_user: root
+  copy:
+    src: "redhat-internal-cert-install-0.1-7.el7.csb.noarch.rpm"
```

Comment:
IMHO this should be removed.

Reply:
Agreed that this is a candidate for an externally added config file.